sim/global-types.ts

```diff
@@ -1253,9 +1253,10 @@ interface FormatsData extends EventMethods {
 	column?: number;
 }
 
-type ModdedFormatsData = FormatsData | Omit<FormatsData, 'name'> & {inherit: true};
+export type FormatList = (FormatData | {section: string, column?: number});
```

Comment:
I believe this is your problem. `export type` changes `global-types` from a global type definition file to a module, which is what's causing all your other crashes.

Reply:
you should remove the word "export" here. Will re-add the error you removed the comment above.